components/UserForm.tsx

```diff
@@ -37,17 +70,21 @@ const UserForm = () => {
 
   return (
     <div className={styles.container}>
-      <form className={styles.form} onSubmit={handleSubmit}>
+      <form className={styles.form} onSubmit={handleSubmit} noValidate>
         <div className={styles.labelInputContainer}>
```

Comment:
Extract input (label + input + error message) into a separate component that receives all required data as props

Reply:
done